webapp/src/app/pages/home/home.component.ts

```
@@ -80,17 +80,18 @@
       for (var course of this.courses) {
         this.Users = [];
-        console.log(' the retrived course url is: ', course);
+        //console.log(' the retrived course url is: ', course);
         this.Users = course.users;
-        // console.log(course.users[0].role.name)
-        //       let userModerator = this.Users.find(
-        //         (user) => user.role.id === 'moderator'
-        //       );
-
-        this.buildCardInfo(course.users[0].userId, course);
+              let userModerator = this.Users.find(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'userModerator' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)